src/combobox/combobox.component.ts

```diff
@@ -58,8 +58,10 @@ import { DocumentService } from "../utils/utils.module";
 				<div
 					*ngIf="type === 'multi' && pills.length > 0"
 					(click)="clearSelected()"
+					(keydown.enter)="clearSelected()"
```

Comment:
nice! didn't know angular can do that

Reply:
I didn't either :D

Reply:
Such a handy feature for things like this! There's a few other modifiers you can stack IIRC